.github/workflows/execution.sh

```diff
@@ -12,7 +12,7 @@ EXECUTION_FILES=(
     ui_testing/testcases/header_tests/test011_rolesandpermissions.py
   )
 
-TEST_REG='test[0-9]{3}'
+TEST_REG='test070'
```

Comment:
you don't run test case, please update  EXECUTION_FILES to select order file only

Reply:
done